apps/src/templates/sectionProgress/VirtualizedSummaryView.jsx

```diff
@@ -71,9 +71,7 @@ class VirtualizedSummaryView extends Component {
         {(rowIndex >= 1 && columnIndex > 0) &&
           <StudentProgressSummaryCell
             studentId={section.students[studentStartIndex].id}
-            studentLevelProgress={studentLevelProgress}
-            stageId={stageIdIndex}
-            scriptData={scriptData}
+            levelsWithStatus={levelsByLessonByStudent[section.students[studentStartIndex].id][stageIdIndex]}
```

Comment:
This line is a lot to work through. I think it makes sense, but wonder if there's a more straightforward way to access the information we need?